README.md

```diff
@@ -33,18 +33,57 @@ Run the `runner.py` file located in the root of the project <br />
 This is the preferred method and allows you to include/exclude tags <br />
 a `~` before the tag name excludes it. <br />
 This is how the tests are run on the CI
-#### Example: `python runner.py --tags smoke --tags ~slow`
+<h4> You MUST specify the environment and product
+#### Example: `python runner.py  --product=EPS-FHIR --env=INT --tags smoke --tags ~slow`
 This will run all tests with the tag `@smoke` but skip any tests tagged with `@slow`
 
 ### Method 2:
 If your IDE supports it, you can directly run the .feature files within `/features`
 <br />
-* Note in some IDEs, you will need to set the `BASE_URL` environment within the behave run configuration rather than the machine
+* Note in some IDEs, you will need to set the `BASE_URL` environment within the behave run configuration rather than the machine.
 
 ### Method 3:
 Run the tests by running `behave` in a command prompt or terminal window.
 * This will run the tests and print the results to console
 
+```
+behave -D product=EPS-FHIR -D env=INT -f behave_cucumber_formatter:PrettyCucumberJSONFormatter -o reports/cucumber_json.json -f
+allure_behave.formatter:AllureFormatter -o allure-results -f pretty features --no-capture --no-capture-stderr --no-skipped --expand --logging-level=DEBUG --tags eps_fhir
+```
+
+change the `env` variable accordingly to either `INT` or `INTERNAL-DEV`.
+If you wish to test a different product i.e. `PFP-APIGREE` then you must change `product=` and `--tags` respectively.
+
 ### Setting the BASE_URL
```

Comment:
I think these steps need to go with setup, if they're needed before being able to run the tests.

They could do with being clearer, too. I wouldn't be confident that I can run these based solely on the readme. Where do we get the base url, and the values to assign to the environment variables, for example?